.chloggen/fix_ca_cofigmap_dots.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. tempostack, tempomonolithic, github action)
+component: operator
```

Comment:
@rubenvp8510 there is no operator component anymore

Reply:
This is a fix that impacts both, monolithic and tempostack. What component should I put here?

Reply:
that could work :D